bower.json

```diff
@@ -24,7 +24,8 @@
     "dist/object-describer.js"
   ],
   "dependencies": {
-    "kubernetes-object-describer": "0.0.10"
+    "kubernetes-object-describer": "0.0.10",
+    "jquery": "~2.1.4"
```

Comment:
can you put jquery in devDependencies, since we only require it for the index.html?